src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/HandlerFactory.java

```diff
@@ -74,6 +74,8 @@ public HandlerFactory() {
         register(TokenTypes.INDEX_OP, IndexHandler.class);
         register(TokenTypes.LITERAL_SYNCHRONIZED, SynchronizedHandler.class);
         register(TokenTypes.LAMBDA, LambdaHandler.class);
+        register(TokenTypes.ANNOTATION_DEF, ClassDefHandler.class);
+        register(TokenTypes.ANNOTATION_FIELD_DEF, AnnotationFieldDefHandler.class);
```

Comment:
http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#ANNOTATION_FIELD_DEF
Since they are more methods than variables, should we base it on `MethodDefHandler` instead?